static/js/compose_ui.js

```diff
@@ -420,10 +423,10 @@ function is_inner_text_formatted(syntax_start, syntax_end, range, selected_text)
 function add_formatting(syntax_start, syntax_end, field, range, text, selected_text) {
     let linebreak_start = "";
     let linebreak_end = "";
-    if(syntax_start[0] === "\n") {
+    if (syntax_start[0] === "\n") {
         linebreak_start = "\n";
     }
-    if(syntax_end[syntax_end.length - 1] === "\n") {
+    if (syntax_end[syntax_end.length - 1] === "\n") {
         linebreak_end = "\n";
```

Comment:
These lint changes should be in the first commit. Every commit should passing `./tools/lint` and `./tools/test-js-with-node --coverage`  independently.